feature-libs/cart/base/components/added-to-cart-toast/added-to-cart-toast.component.ts

```diff
@@ -54,7 +54,7 @@ export class AddedToCartToastComponent implements OnInit, OnDestroy {
     this.headerElement = this.winRef.document.querySelector('header');
 
     if (!this.customClass) {
-      this.customClass = 'cx-added-cart-toast';
+      this.customClass = 'cx-added-to-cart-toast';
```

Comment:
Some inconsistency in naming, in selectors I see `added-to-cart` and i.e. in constants `ADD_TO_CART`. Let's decide on one and apply it on all cases